README.md

```
@@ -110,147 +177,138 @@ npm run dev
+
+#### Project Structure
 
-- **Current Phase:** Cleanup + parity hardening
-- **Status:** Core runtime on Tauri, repository cleanup still in progress
-
-For archived migration history, see [Electron to Tauri Migration Status](docs/electron-old/electron-to-tauri-migration-status.md). For the current cleanup snapshot, use [Tauri Cleanup Status (2026-03-06)](docs/handoffs/tauri-cleanup-status-2026-03-06.md).
-
-## Tech Stack
-
-### Application
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)